bridge/engine/EngineMixer.cpp

```diff
@@ -2190,10 +2380,12 @@ void EngineMixer::forwardAudioRtpPacket(IncomingPacketInfo& packetInfo, uint64_t
                 ssrc = rewriteMapItr->second;
             }
 
+            const bool isForwardSsrc = !ssrcRewrite;
```

Comment:
I would prefer if there was another method to call to separate this. Also this variable is not really helping I think.